src/viperleed/gui/measure/classes/settings.py

```diff
@@ -94,7 +111,200 @@ class TooManyDefaultSettingsError(DefaultSettingsError):
     """Exception raised when too many default settings files were found."""
 
 
-class ViPErLEEDSettings(ConfigParser):
+class AliasConfigParser(ConfigParser):
+    """A ConfigParser that checks for aliases when getting values.
+
+    To load aliases, the prepare_aliases() method must be called with
+    the appropriate cls_name argument. To perform an automatic check for
+    aliases when getting an option, the get() method and its derivatives
+    must be used. Getting values via the __getitem__ method (e.g.,
+    self[sec][opt], has_option()) will not check for aliases.
+    """
+
+    def __init__(self, *args, cls_name='', **kwargs):
+        """Initialise AliasConfigParser instance."""
+        super().__init__(*args, **kwargs)
+        self._aliases = {}
+        self._fallback = defaultdict(dict)
+        self._cls_name = ''
+        self.prepare_aliases(cls_name)
+
+    def get(self, section, option, *, raw=False, vars=None, fallback=_UNSET):
+        """Get an option value for a given section or an alias of it.
+
+        This method will attempt to get a value from self first. If this
+        fails, the loaded aliases are checked. If this fails too,
+        fallback is used. If the gotten value is an empty string, the
+        aliases fallback values are checked.
+
+        Parameters
+        ----------
+        section : str
+            The section key.
+        option : str
+            The option key.
+        raw : bool, optional
+            Whether interpolations in the return value should be
+            expanded. If False, interpolations are expanded. Default
+            is False.
+        vars : dict, optional
+            If provided, the option is looked up in vars before
+            checking section. Default is None.
+        fallback: object, optional
+            If provided, fallback is the return value if getting the
+            option failed. Default is _UNSET.
+
+        Returns
+        -------
+        value : str
+            The read value from the loaded settings.
+
+        Raises
+        ------
+        NoSectionError
+            If the settings section is missing.
+        NoOptionError
+            If the settings option is missing.
+        """
+        value = _UNSET
+        try:
+            value = super().get(section, option, raw=raw, vars=vars)
+        except (NoSectionError, NoOptionError) as err:
```

Comment:
```suggestion
        except (NoSectionError, NoOptionError):
```

Since you don't use `err` for anything other than re-raising, you can skip the "as" part here, and just use a bare `raise` instead of `raise err`.